website/docs/automation/controlplanes/crossplane/resources.md

```diff
@@ -3,106 +3,127 @@ title: "Managed Resources"
 sidebar_position: 20
 ---
 
-By default the catalog component in the sample application uses a MySQL database running as a pod in the EKS cluster. In this lab, we'll provision an Amazon RDS database for our application using Kubernetes custom resources to specify the desired configuration required by the workload.
+By default the **Carts** component in the sample application uses a DynamoDB local instance running as a pod in the EKS cluster called ```carts-dynamodb```. In this section of the lab, we'll provision an Amazon DynamoDB cloud based table for our application using Crossplane managed resources and point the **Carts** deployment to use the newly provisioned DynamoDB table instead of the local copy.
 
-Let's explore the various Crossplane resources that we'll create. The first is an EC2 security group that will be applied to control access to the RDS database, which is done with a `ec2.aws.crossplane.io.SecurityGroup` resource:
+![ACK reconciler concept](./assets/Crossplane-desired-current-ddb.png)
 
-```file
-manifests/modules/automation/controlplanes/crossplane/managed/rds-security-group.yaml
-```
+The AWS Java SDK in the **Carts** component is able to use IAM Roles to interact with AWS services which means that we do not need to pass credentials, thus reducing the attack surface. In the EKS context, IRSA allows us to define per pod IAM Roles for applications to consume. To leverage IRSA, we first need to:
 
-:::info
+- Create a Kubernetes Service Account in the Carts namespace
+- Create an IAM Policy with necessary DynamoDB permissions
+- Create an IAM Role in AWS with the above permissions
+- Map the Service Account to use the IAM role using Annotations in the Service Account definition.
 
-The EC2 security group above allows any traffic from the CIDR range of the VPC used by the EKS cluster. This has been done to keep the example clear and understandable. A more secure approach would be to use [Security Groups for Pods](../../../networking/security-groups-for-pods/index.md) to allow traffic from specific pods.
+Fortunately, we have a handy one-liner to help with this process. Run the below:
 
-:::
+```bash
 
-Next, we want the RDS database to use the private subnets in our VPC. We'll create a `database.aws.crossplane.io.DBSubnetGroup` which selects the appropriate subnet IDs:
+$ eksctl create iamserviceaccount --name carts-crossplane \
+  --namespace carts --cluster $EKS_CLUSTER_NAME \
+  --role-name carts-dynamodb-role \
```

Comment:
Please use EKS_CLUSTER_NAME in the role name https://github.com/aws-samples/eks-workshop-v2/blob/61ba415ebfe731ee9cc7d7e662e9904494a5b8c9/website/docs/automation/controlplanes/ack/provision-resources.md?plain=1#L22

Reply:
EKS_CLUSTER_NAME used for the role name parameter